src/frontend/Komponenter/Behandling/Inngangsvilkår/vilkår.ts

```diff
@@ -28,9 +28,16 @@ export interface IVilkårGrunnlag {
     dokumentasjon?: IDokumentasjonGrunnlag;
     harAvsluttetArbeidsforhold: boolean;
     harKontantstøttePerioder?: boolean;
+    kontantstøttePerioder?: KsPeriode[];
     behandlingOpprettet?: string;
 }
 
+export interface KsPeriode {
```

Comment:
```suggestion
export interface KontantstøttePeriode {
```

Mindre mental overhead